std/asyncio/system/StdioConfig.hx

```diff
@@ -0,0 +1,45 @@
+package asyncio.system;
+
+import asyncio.filesystem.File;
+import asyncio.filesystem.FileOpenFlag;
+import asyncio.filesystem.FilePath;
+
+/**
+	This enum allows to configure IO channels of a process being created with functions
+	like `asyncio.system.Process.open` and such.
+	@see asyncio.system.Process
+**/
+enum StdioConfig {
+	/**
+		Create a unidirectional pipe for IO channel.
+		The child process will be able to read from the pipe, while the parent
+		process will be able to write to the pipe.
+		This is the default behavior for stdin.
+	**/
+	PipeRead;
+	/**
+		Create a unidirectional pipe for IO channel.
+		The child process will be able to write to the pipe, while the parent
+		process will be able to read from the pipe.
+		This is the default behavior for stdout and stderr.
+	**/
+	PipeWrite;
+	/**
+		Use the corresponding IO stream of the parent process.
+		For example if `Inherit` is used for stdin of the child process, then stdin
+		of the parent process will be used.
+	**/
+	Inherit;
+	/**
+		Connect IO channel to `/dev/null` on unix-like systems and to `NUL` on windows.
+	**/
+	Ignore;
+	/**
+		Use specified file as a source and/or a target for IO.
+	**/
+	File(path:FilePath, flags:FileOpenFlag<Dynamic>);
+	/**
+		Use an opened file as a source and/or a target for IO.
+	**/
+	OpenedFile(file:File);
+}
```

Comment:
Bidirectional pipes are missing.

Reply:
Wouldn't that impede portability?

Reply:
By that logic we shouldn't have unix sockets and a bunch of other operations. It will not work on some platforms, but we should still make it available where it works.

Reply:
Yes, they are missing because of no good counterpart on all systems.
I was thinking about special modules like `asyncio.system.Posix` and `asyncio.system.Win` for such things. But I guess this approach wouldn't work for enums anyway.

Reply:
Added `PipeReadWrite`